config/locales/en.yml

```diff
@@ -7,6 +7,7 @@ en:
     invalid_confirmation_code: Invalid confirmation code.
     messages:
       improbable_phone: Phone number is invalid. Please make sure you enter a 10-digit phone number.
+      weak_password: not strong enough. Consider adding a number, symbols or more letters to make it stronger. %{feedback}
```

Comment:
I just noticed that we're including the gem's feedback text, which says `This is similar to a commonly used password. Add another word or two. Uncommon words are better.` This makes our custom text feel redundant, no? Should we choose or the other?